load_confounds.py

```diff
@@ -235,6 +279,15 @@ def load_confounds(
         "derivatives" global signal and derivative (2 parameters)
         "full" global signal + derivatives + quadratic terms + power2d derivatives (4 parameters)
 
+    compcor : string,optional
+    	Type of confounds extracted from a component based noise correction method
+    	"anat" noise components calculated using anatomical compcor
+    	"temp" noise components calculated using temporal compcor
+    	"full" noise components calculated using both temporal and anatomical
+
+   	n_compcor : int, optional
+   		The number of noise components to be extracted. If -1 all noise components are used.
```

Comment:
It looks to me like the indentation is not consistent with the rest of the docs, but I may be wrong on that one.